contracts/Insurance.sol

```diff
@@ -68,11 +69,11 @@ contract Insurance is IInsurance, Ownable, SafetyWithdraw {
         updatePoolAmount();
         InsurancePoolToken poolToken = InsurancePoolToken(token);
 
-        // tokens to mint = (pool token supply / collateral holdings) * collaterael amount to stake
+        // tokens to mint = (pool token supply / collateral holdings) * collateral amount to stake
```

Comment:
How are we defining collateral holdings? Is collateral holdings just collateral amount (public), or is it collateral amount (public) + buffer? 

It should be the second one